internal/storage/system/migrations.go

```diff
@@ -239,6 +240,66 @@ func GetMigrator(db bun.IDB, options ...migrations.Option) *migrations.Migrator
 				})
 			},
 		},
+		migrations.Migration{
+			Name: "Create buckets table",
+			Up: func(ctx context.Context, db bun.IDB) error {
+				return db.RunInTx(ctx, &sql.TxOptions{}, func(ctx context.Context, tx bun.Tx) error {
+					_, err := tx.ExecContext(ctx, `
+						create table _system.buckets (
```

Comment:
You should initialize the added_at column with the creation date of the first ledger created on this bucket